build/toastr.css

```diff
@@ -178,24 +188,41 @@ button.toast-close-button {
     padding: 8px 8px 8px 50px;
     width: 11em;
   }
+  #toast-container > div.rtl {
+    padding: 8px 50px 8px 8px;
+  }
   #toast-container .toast-close-button {
     right: -0.2em;
     top: -0.2em;
   }
+  #toast-container .rtl .toast-close-button {
+    left: -0.2em;
+    right: 0.2em;
```

Comment:
Already tops from the [non-rtl version](https://github.com/CodeSeven/toastr/pull/439/files#diff-cc044eb2261831f76b5653c793d7ecd8R196).